Formula/lib/libadwaita.rb

```diff
@@ -25,6 +25,8 @@ class Libadwaita < Formula
     sha256 x86_64_linux:   "9c9ed668d719bab47cc21c920d3214de8c81b9447232ba8f0bce968053a41b05"
   end
 
+  depends_on "appstream" => [:build, :test]
```

Comment:
this needs to be runtime dep?
```suggestion
  depends_on "appstream"
```

Reply:
Thanks @chenrui333, great catch!